server/model/taggedBailDaysForm.ts

```diff
@@ -0,0 +1,41 @@
+import { Adjustment } from '../@types/adjustments/adjustmentsTypes'
+import AbstractForm from './abstractForm'
+import ValidationError from './validationError'
+
+export default class TaggedBailDaysForm extends AbstractForm<TaggedBailDaysForm> {
+  days: number
+
+  isEdit: boolean
+
+  adjustmentId?: string
+
+  toAdjustment(adjustment: Adjustment): Adjustment {
+    return { ...adjustment, days: this.days }
+  }
+
+  async validation(): Promise<ValidationError[]> {
+    if (!this.days) {
+      return [
+        {
+          text: 'Enter the number of days for the tagged bail',
+          fields: ['days'],
+        },
+      ]
+    }
+    if (this.days <= 0) {
+      return [
+        {
+          text: 'Enter a whole number for the number of days on tagged bail',
```

Comment:
There needs to be a further investigation of this validation criteria and message in a subsequent ticket